model/realtime.go

```diff
@@ -37,3 +38,7 @@ type Message struct {
 	Data interface{} `json:"data"`
 	ID   string      `json:"id"` // the request id
 }
+
+type LiveQueryOptions struct {
+	SkipInitial bool `json:"skipInitial"`
```

Comment:
Wasn't this supposed to be change only. Lets use a consistent nomenclature across

Reply:
I think no. They are 2 diff things. The server has nothing to do with `changesOnly`. That is handled entirely on the client. The server just needs to know whether to send the initial or not. (Internally, in the clients, I am setting the values of both to be the same)

Reply:
Cool. Works for me.